### ISSUE TITLE
Google Classroom doesn't log in

### ISSUE THREAD

**Issue by BeeeWall** [microg/android_packages_apps_GmsCore#728]
All accounts show up, but when I select one and hit "ok" it gives an error. Using an old version (see below) and then updating allows me to view the current account, but not to switch accounts. 

[Logcat](https://pastebin.com/raw/SwUEFgrQ) with Google Classroom version 5.1.062.05.40 (I'm pretty sure you can ignore the Launcher.Utilities and SDM messages). 

The last working version of Classroom I can find is 3.5.212.04.40 (however, I cannot confirm that all versions after have the same errors, as I've only gotten a log with the latest version, just that the version after this stopped working). 

TL;DR:
- Google Classroom doesn't log in correctly
- Classroom version: 5.1.062.05.40
- Last working Classroom version: 3.5.212.04.40
- microG version: 0.2.6.15090-dirty-167
- logcat: https://pastebin.com/raw/SwUEFgrQ

**Comment by BeeeWall** [comment]
Works now. Not sure if it was fixed or just wasn't a microG issue.

**Comment by swedneck** [comment]
I am affected by this as well, maybe it'll fix itself :P

**Comment by happycoder97** [comment]
I'm affected by this.
microG version: 0.2.10.19420
Classroom version: 6.5.222.04.30

**Comment by happycoder97** [comment]
PS: You won't be able to switch accounts or use multiple accounts in the above workaround
